splinter/driver/webdriver/__init__.py

```diff
@@ -97,21 +97,17 @@ def next(self):  # NOQA: A003
         next_handle = self._browser.driver.window_handles[next_index]
         return Window(self._browser, next_handle)
 
-    def is_current():
```

Comment:
Using decorators here is good, but I'd prefer if you split them into a separate PR. They're not related to the rest of the changes.

Reply:
Reverted file; will submit separately.